### PR TITLE
Add Nov. Stable Release APIs to "Overview of APIs"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2913&tabs=winrtcsharp#custom-scheme-registration

AB#47458181

### CODE REVIEW

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 1]
```diff
@@ -970,6 +970,33 @@ WebView2 provides functionality to handle the JavaScript function `window.open()
 
 ---
 
+
+#### todo: add link to the above tab-set?  Nov 2023 incoming
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:974]: I don't think we need to add to this one. It's a property of a class that's already existing

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 2]
```diff
@@ -1539,6 +1566,42 @@ The `CustomSchemeRegistration` allows registration of custom schemes in WebView2
...
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2CustomSchemeRegistration` Class:
+    * [CoreWebView2CustomSchemeRegistration.AllowedOrigins Property](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration.allowedorigins)
+    * [CoreWebView2CustomSchemeRegistration.SchemeName Property](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration.schemename)
+
+* `CoreWebView2EnvironmentOptions` Class:
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations)
+
+##### [WinRT/C#](#tab/winrtcsharp)
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:1582]: for this one, only update WinRT tab to be the same as the .NET tab: i.e.

* [CoreWebView2CustomSchemeRegistration Class](microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration)
* `CoreWebView2EnvironmentOptions` Class:
    * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#customschemeregistrations)

**Review by victorhuangwq — CHANGES_REQUESTED** [review]
Do not update the newwindowrequested api, but do update the winrt tab of custom scheme registration

File: microsoft-edge/webview2/concepts/overview-features-apis.md [hunk 3]
```diff
@@ -970,6 +970,33 @@ WebView2 provides functionality to handle the JavaScript function `window.open()
 
 ---
 
+
+#### todo: add link to the above tab-set?  Nov 2023 incoming
+
+* Added source frame info to the `NewWindowRequested` event arguments, to identify the source of the request:
```
Comment (peiche-jessica) [inline microsoft-edge/webview2/concepts/overview-features-apis.md:976]: If we are not calling out other properties from the NewWindowRequestedEventArgs, we don't need to explicitly call the OriginalSourceFrameInfo either.

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:974]: removed temp section

**Review by peiche-jessica — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/concepts/overview-features-apis.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/overview-features-apis.md:976]: removed temp section

**Review by peiche-jessica — APPROVED** [review]

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**